python/ray/tests/test_memstat.py

```diff
@@ -48,24 +59,25 @@ def count(memory_str, substr):
 
 
 def test_driver_put_ref(ray_start_regular):
-    info = memory_summary()
+    info = memory_summary(ray_start_regular["redis_address"])
```

Comment:
Can we just do `address = ray_start_regular["redis_address"]` and pass them instead of copying all?